frontend/public/components/storage-class.tsx

```diff
@@ -1,27 +1,34 @@
 import * as React from 'react';
```

Comment:
@logonoff I think we can also remove all `react` namespace imports under `frontend/public`

Reply:
the `React` namespace import is kept due to the usage of `React.FC`. 

in my types/react 18 upgrade I plan to replace these imports with named imports to avoid importing the entirety of react every time we want to use a type

Reply:
> I plan to replace these imports with named imports to avoid importing the entirety of react every time we want to use a type

:+1: when doing that, we can also add ESLint rule that disallows `import * as React` namespace imports.

We can also consider using `import type` instead of `import` specifically for TS types :smiley: that would make TS compiler happy for sure. So when we import both code and types via single import statement, it could be broken down into one `import` and one `import type` along with appropriate ESLint import rule.

Reply:
I guess this comment can be disregarded for now.

Reply:
Right, for now it's not an issue.